src/Storages/AwsS3Storage.php

```diff
@@ -67,8 +64,9 @@ public function getBackup(Client $sapi, string $path): Backup
             if ($e->getAwsErrorCode() === 'NoSuchKey') {
                 throw new UserException(
                     sprintf(
-                        'Backup with ID "%s" was not initialized for this KBC project',
-                        $this->backupId
+                        'Backup path "%s" was not initialized in the bucket "%s".',
```

Comment:
Nebolo by lepsie `Backup path ... not found in the bucket` ... takto z hlasky neviem co znamena `initialized`.

Reply:
jj může být